model/clusters_mgmt/v1/break_glass_credential_type.model

```diff
@@ -28,16 +31,22 @@ class BreakGlassCredential {
 	// Status is the status of this credential
 	Status BreakGlassCredentialStatus
 
+    // Additional information about the current state of this credential
+    StatusDetails BreakGlassCredentialStatusDetails
```

Comment:
np; indentation

Reply:
this must be my vscode setting cause it's indented properly there 😂 I'll fix, thanks